data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -1535,12 +1535,60 @@
     "//": "calculation based on https://www.youtube.com/watch?v=5ZrfNAHDjWU; 1 matchbox contains 30 mg phosphorus and 1 unit of phosphor is 100 mg",
     "components": [ [ [ "survival_match", 1 ], [ "ref_matches", 3 ], [ "matches", 5 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "chem_calcium_phosphate",
+    "charges": 8,
+    "batch_time_factors": [ 95, 1 ],
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "difficulty": 3,
+    "skill_used": "chemistry",
+    "time": "10 m",
+    "qualities": [ { "id": "CHEM", "level": 2 } ],
+    "book_learn": [ [ "textbook_gaswarfare", 2 ], [ "textbook_chemistry", 3 ], [ "adv_chemistry", 3 ], [ "textbook_anarch", 3 ] ],
+    "tools": [ [ [ "surface_heat", 20, "LIST" ] ] ],
```

Comment:
What process is this supposed to be? I'm not aware of a process where heating bone meal on a stove is productive.